pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -128,4 +128,26 @@ under the License.
             <resolver>org.apache.hawq.pxf.plugins.gemfirexd.GemFireXDResolver</resolver>
         </plugins>
     </profile>
+	<profile>
+		<name>JSON</name>
+		<description>A profile for JSON data, one JSON record per line
+		</description>
+		<plugins>
+			<fragmenter>org.apache.hawq.pxf.plugins.hdfs.HdfsDataFragmenter</fragmenter>
+			<accessor>org.apache.hawq.pxf.plugins.json.JsonAccessor</accessor>
+			<resolver>org.apache.hawq.pxf.plugins.json.JsonResolver</resolver>
+		</plugins>
+	</profile>
+	<profile>
+		<name>JSON-PP</name>
```

Comment:
what does PP stand for?
Other profiles use CamelCase as their naming standing, like HdfsTextSimple or HiveRc - perhaps change it to something like that?